lib/Camp/Master.pm

```diff
@@ -2843,6 +2851,21 @@ sub db_version_pg {
     return $major_minor;
 }
 
+sub db_version_mysql {
+    my %arg = @_;
+
+    # Some examples of $raw output:
+    # 5.7.15-9
+    # 10.1.20-MariaDB
```

Comment:
@hexfusion We don't really support MariaDB, do we? At any rate given its totally different version numbers it seems weird to treat it as the same thing here.

What versions of MySQL and/or MariaDB have you tested these changes with?